data/react-cheat-sheet.json

```diff
@@ -0,0 +1,86 @@
+{
+    "id": "reactjs-cheat-sheet",
+    "title": "রিয়েক্ট JS টুকিটাকি",
+    "slug": "reactjs-cheat-sheet",
+    "description": "রিয়েক্ট JS এর আদ্যোপান্ত",
+    "colorPref": "#0288d1",
+    "contents": [
+      {
+        "title": "রিয়েক্ট JS এ 'Hello world' প্রিন্ট করা",
+        "items": [
+          {
+            "defination": "react কে ইম্পোর্ট(ES6 কনসেপ্ট) করে নেয়া",
```

Comment:
ধন্যবাদ ভাই। এখানে ছোট্ট একটা সমস্যা আছে। স্পেলিং টা হবে `definition` not `defination`। এইটার জন্য কাজ করবে না এই ফাইল টা। একটু যদি ঠিক করে আবার পুশ করেন তাহলে খুব শীঘ্রই এটা merge হবে।